languages/javascript/config.json

```diff
@@ -27,6 +27,12 @@
         "concepts": ["booleans-basic"],
         "prerequisites": []
       },
+      {
+        "slug": "arrays",
+        "uuid": "25cb0707-44f8-4800-b993-3fcb2b6d9f61",
+        "concepts": ["booleans-basic"],
+        "prerequisites": ["arrays-basic"]
```

Comment:
The exercise doesn't depend on the arrays-basic concept as it aims to teach that :) I do think there is a prerequisite on working with integers, as they are used as the type stored in the array.